src/tinykeys.ts

```diff
@@ -214,7 +214,7 @@ export function createKeybindingsHandler(
  * })
  * ```
  */
-export default function keybindings(
+function keybindings(
```

Comment:
want to just name this like:

```ts
export function tinykeys(...) {...}
```

Reply:
Thanks for review!

Fixed from this commit.
https://github.com/jamiebuilds/tinykeys/pull/161/commits/629b037a271ed4de160ae232825c9ee335b009e4